### ISSUE TITLE
workshop todo

### ISSUE THREAD

**Issue by kushalkolar** [kushalkolar/fastplotlib#63]
stuff to try and finish before the workshop:

- [x] #91 
- [x] docs
- [x] demo with the following, a gridplot and an ImageWidget
  - gridplot:
    - behavior side and front, heatmap, stacked lineplot, single lineplot which shows only the current selection, stacked lineplot where each line is a trial of the selected component, PCA, downstream analysis
  - imagewidget:
    - mcorr, RCM, etc. with contours that interact with the stacked lineplot, single lineplot and heatmap
- [x] high level interactivity, #93 
  - #39 
- [x] autoscale camera for data w.r.t. data bounds
  - done in #80 
- [x] See how "big_camera" settings work as the default with not-so-large near & far planes, #62 
  - setting `PlotArea.controller.distance = 0` seems to have fixed it, it was merged in #80 
- [ ] `Heatmap` with tiling `pygfx.Image`, https://github.com/pygfx/pygfx/issues/360 
- [x] API renaming where necessary
- [x] better handling of `color` args to `Graphic` objects
  - #78 
  - need to make new PR after #39 to make this work with Collections, https://github.com/kushalkolar/fastplotlib/issues/76#issuecomment-1358903126 
- [x] `LineStack`
- [x] Make LineCollection better
- [x] very long LineGraphic object that changes position with a slider. Can just make a widget which takes an existing IntSlider as an argument and automatically connects the observe to itself. Exclude in scene bbox calculation, or put it in a diff scene. Make it an overlay of 2 lines, thick with low alpha and thin with high alpha on top. 
- [ ] `LinearSelector` and/or `RectangleSelector`, see for ideas: #62
- [ ] histogram widget for images (needs RectangleSelector) - we have vmin vmax sliders for now
